api/client/samples/crop_models/sao_paulo_sugar_content/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3628 @@
+{
```

Comment:
<p>Just one language tweak: since it is all in sample prediction, it is more clear to say <code>fitted yield</code> than <code>predicted yield</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
<p>Changed into "within-sample predicted yield"</p>